src/braille/internal/braille.cpp

```diff
@@ -310,6 +316,292 @@ namespace mu::engraving {
 #define BRAILLE_UP_BOW                  QString("<'")
 #define BRAILLE_LEFT_HAND_PIZZICATO     QString("_>")
 
+std::string Braille_UpperNumbers[] = { "245", "1", "12", "14", "145",
+                                       "15", "124", "1245", "125", "24" };
+
+std::string Braille_LowerNumbers[] = { "356", "2", "23", "25", "256",
+                                       "26", "235", "2356", "236", "35" };
+
+std::string Braille_LyricLineIndicator = "56-23";
+
+std::string Braille_NumIndicator = "3456";
+
+std::string getBraillePattern(std::string dots)
```

Comment:
For all these functions with `std::string` or `QString` as a parameter: shouldn't it be `const std::string&` or `const QString&`?